src/v3/src/mocks/scenario/securityquestion-enroll-mfa.ts

```diff
@@ -56,12 +56,28 @@ scenario('securityquestion-enroll-mfa', (rest) => ([
     );
   }),
 
-  // send security question data
+  // send security question data or return invalid answer for custom or predefined question when answer < 4 characters
   rest.post('*/idp/idx/challenge/answer', async (req, res, ctx) => {
-    const { default: body } = await import('../response/idp/idx/challenge/answer/securityquestion-enroll-mfa.json');
+    const request = req.body as Record<string, any>;
+    const questionKey = request.credentials?.questionKey;
+    const answer = request.credentials?.answer;
+    let response = null;
+    let responseStatus;
+    if (answer?.length < 4) {
+      if (questionKey === 'custom') {
+        response = (await import('../response/idp/idx/challenge/answer/enroll-security-question-custom-with-error.json')).default;
+      } else {
+        response = (await import('../response/idp/idx/challenge/answer/enroll-security-question-predefined-with-error.json')).default;
+      }
+      // responseStatus = 401;
+      responseStatus = 200; // error from auth-js causes breakage when setting 401 status on this flow
```

Comment:
Ah, I found out a little bit more about why this is a 200 instead of 401. Will DM you with link to relevant Slack thread